test/base/ByteBufferTest.ooc

```diff
@@ -9,20 +9,6 @@
 use base
 use unit
 
-CustomAllocator: class extends AbstractAllocator {
```

Comment:
Didn't you implement this because you needed it for something specific @sebastianbaginski ?